src/database/schemas.ts

```diff
@@ -142,4 +143,22 @@ export default class Database {
 
     return formatted;
   }
+  
+  static async deleteObject(objectPath: string, type: string): Promise<void> {
```

Comment:
Instead of objectPath, do we want to maintain the pattern of specifying schema and object name as separate values?